### PR TITLE
opendatahub-io/ai-edge: RHOAIENG-2382 - Explore and setup initial nightly check

### PR DESCRIPTION
**Pull request by MarianMacik**
**JIRA:** [RHOAIENG-2382](https://issues.redhat.com/browse/RHOAIENG-2382)

/cc @LaVLaS

### CODE REVIEW

File: ci-operator/config/opendatahub-io/ai-edge/opendatahub-io-ai-edge-main.yaml [hunk 1]
```diff
@@ -46,6 +46,36 @@ tests:
         requests:
           cpu: 500m
     workflow: hypershift-hostedcluster-workflow
+- as: test-ai-edge-periodic
+  cron: 0 * * * *
```
Comment (MarianMacik) [inline ci-operator/config/opendatahub-io/ai-edge/opendatahub-io-ai-edge-main.yaml:50]: This is just for testing. We have to first merge it to check that the Slack notifications work.

File: core-services/prow/02_config/opendatahub-io/ai-edge/_prowconfig.yaml [hunk 2]
```diff
@@ -1,3 +1,16 @@
+slack_reporter_configs:
+  opendatahub-io/ai-edge:
+    channel: '#mmacik-test'
```
Comment (MarianMacik) [inline core-services/prow/02_config/opendatahub-io/ai-edge/_prowconfig.yaml:3]: This will be changed in the subsequent PR.

**Comment by MarianMacik** [comment]
> /lgtm

@LaVLaS your lgtm didn't work for some reason, can you retry?

**Comment by LaVLaS** [comment]
/lgtm

I think the previous attempt was ignored due to prow jobs being delayed

**Pull request merged**